packages/core/src/components/QueryBuilder/QueryBuilder.tsx

```diff
@@ -33,32 +33,32 @@ export { staticClasses as queryBuilderClasses };
 export type HvQueryBuilderClasses = ExtractNames<typeof useClasses>;
 
 export interface HvQueryBuilderProps {
-  attributes?: Record<string, Attribute>;
+  attributes?: Record<string, HvQueryBuilderAttribute>;
   /**
    * The query rules operators by attribute type and combinator.
    */
-  operators?: Record<string, QueryOperator[]>;
+  operators?: Record<string, HvQueryBuilderQueryOperator[]>;
   /**
    * The query combinators operands.
    */
-  combinators?: QueryCombinator[];
+  combinators?: HvQueryBuilderQueryCombinator[];
   /**
    * The initial query representation.
    */
-  query?: Query;
+  query?: HvQueryBuilderQuery;
   /**
    * Callback fired when query changes.
-   * @param {Query} value - the query representation.
+   * @param {HvQueryBuilderQuery} value - the query representation.
```

Comment:
TS already takes care of this
```suggestion
   * @param value - the query representation.
```